reference/docs-conceptual/SecretManagement/get-started/using-secretstore.md

```diff
@@ -0,0 +1,65 @@
+---
+description: How to install and use the SecretManagement and SecretStore modules
+ms.date: 05/24/2022
+title: Get started with the SecretStore module
+---
+# Get started with the SecretStore module
+
+The **SecretManagement** and **SecretStore** modules are available from the PowerShell Gallery and
+can be installed using **PowerShellGet** commands.
+
+```powershell
+Install-Module Microsoft.PowerShell.SecretManagement
+Install-Module Microsoft.PowerShell.SecretStore
+```
+
+Once you have installed the modules you can load the modules and begin using creating new secrets.
+
+```powershell
+Import-Module Microsoft.PowerShell.SecretManagement
+Import-Module Microsoft.PowerShell.SecretStore
+```
+
+## Create a vault and add a secret
+
+First you must register the vault. The **Name** parameter is a friendly name and can be anything you
+choose.
+
+```powershell
+Register-SecretVault -Name SecretStore -ModuleName Microsoft.PowerShell.SecretStore -DefaultVault
+```
+
+Now you can create a secret.
+
```

Comment:
Not sure if it would make sense here to specify the types of secrets that the vault accepts

Reply:
What types are supported? I can add it.

Reply:
Added